Formula/qtads.rb

```diff
@@ -28,21 +28,37 @@ class Qtads < Formula
   depends_on "qt@5"
   depends_on "sdl2"
 
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
+
   def install
     sdl_sound_include = Formula["sdl_sound"].opt_include
     inreplace "qtads.pro",
       "$$T3DIR \\",
       "$$T3DIR #{sdl_sound_include}/SDL \\"
 
     qt5 = Formula["qt@5"].opt_prefix
-    system "#{qt5}/bin/qmake", "DEFINES+=NO_STATIC_TEXTCODEC_PLUGINS"
+    args = ["DEFINES+=NO_STATIC_TEXTCODEC_PLUGINS"]
+    args << "PREFIX=#{prefix}" unless OS.mac?
+
+    system "#{qt5}/bin/qmake", *args
```

Comment:
```suggestion
    system Formula["qt@5"].opt_bin/"qmake", *args
```

Reply:
Actually I'm not really sure why this call isn't just `system "qmake"` - `qmake` is in the path just like any other binary.  I'll try this first as it would certainly be the simplest.

Reply:
Just using `system "qmake"` worked as expected.  This should be good to go!